client/state/sites/hooks/use-billing-history.ts

```diff
@@ -13,7 +13,7 @@ const fetchPastBillingTransactions = () =>
 		apiVersion: '1.3',
 	} );
 
-export const usePastBillingTransactions = () => {
+export const usePastBillingTransactions = ( isInSignup: boolean ) => {
```

Comment:
If the only purpose of `isInSignup` is to disable this hook, I think that a disabled parameter would be more semantic and reusable.

```suggestion
export const usePastBillingTransactions = ( disabled: boolean ) => {
```